src/cljs/ataru/virkailija/editor/view.cljs

```diff
@@ -88,23 +87,23 @@
    [form-controls]])
 
 (defn editor-name []
-  (let [form              (subscribe [:editor/selected-form])
-        new-form-created? (subscribe [:state-query [:editor :new-form-created?]])
-        form-name         (reaction (:name @form))]
+  (let [form              @(subscribe [:editor/selected-form])
```

Comment:
This could have similar issue as in my earlier comments, but not sure. Plz test at least.